openedx/tests/completion_integration/test_models.py

```diff
@@ -69,7 +69,7 @@ def test_changed_value(self):
         self.assertEqual(models.BlockCompletion.objects.count(), 1)
 
     def test_unchanged_value(self):
-        with self.assertNumQueries(1):  # Get
+        with self.assertNumQueries(3):  # Get, 2 * savepoints
```

Comment:
Shouldn't this be `1*SELECT +  2*DUMMY`, as it looks that we have two savepoints above already :thinking: